docs/Modules.md

```diff
@@ -57,7 +57,7 @@ Here's a list of modules bundled with the game by default (this line-up will cha
 * Core - mandatory content needed for normal game launch
 * [CoreAssets](https://github.com/Terasology/CoreAssets) - base assets extracted from Core
 * [CoreBlocks](https://github.com/Terasology/CoreBlocks) - base blocks extracted from Core
-* CoreSampleGameplay - gameplay front for Core - allows modules to depend on Core without the default starting inventory
+* [CoreSampleGameplay](https://github.com/Terasology/CoreSampleGameplay) - default gameplay with a few core modules enabled
```

Comment:
What is this file used for? I don't think it's up to date, is it?

Reply:
I've done best effort to keep `Modules.md` in sync manually with the omega lineup, yeah, but that's a poor approach. More automation would be better (like the module site)